src/app/app.component.ts

```diff
@@ -308,10 +330,10 @@ export class AppComponent {
   }
 
   // Show the chat widget on: 'login', 'browse/library', or 'browse/groups/public'
-  private showChatWidget(eventUrl: string): boolean {
+  private shouldShowChat(eventUrl: string): boolean {
       if (eventUrl.indexOf('browse/library') > -1 ||
           eventUrl.indexOf('browse/groups/public') > -1 ||
-          eventUrl.indexOf('login') > -1) {
+          eventUrl.indexOf('/login') > -1) {
```

Comment:
small bonus change. I don't think it was causing an issue here but if the string 'login' appeared anywhere in the url, then the widget would show. I think adding a `/` in front of `login` will help cut down the possibility unintended matches in the future.